it-test/functions.ts

```diff
@@ -70,7 +70,7 @@ export function getServiceName(part: Part) {
 }
 
 function procArgs() {
-    return _.takeRightWhile(process.argv, i => i.startsWith('--'));
```

Comment:
Perhaps change it to `return _.takeRightWhile(process.argv, i => i !== 'ittest');` ?